packages/react/src/reactrouter.tsx

```diff
@@ -184,3 +285,22 @@ export function withSentryRouting<P extends Record<string, any>, R extends React
   return WrappedRoute;
 }
 /* eslint-enable @typescript-eslint/no-explicit-any, @typescript-eslint/no-unsafe-member-access */
+
+function getActiveRootSpan(): Span | undefined {
```

Comment:
might be worth lifting this utility out

Reply:
yeah, but here it's a bit special because it also takes the `activeTransaction` into consideration!